internal/metrics/metrics.go

```diff
@@ -30,15 +39,47 @@ func New() Metrics {
 			Help: "Number of existing KMMO Modules",
 		},
 	)
-	return &metrics{kmmoResourcesNum: kmmoResourcesNum}
+	completedStages := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: completedKMMOStageQuery,
+			Help: "For a given kmmo,namespace, stage(device-plug, driver-container, build), 1 if the stage is completed, 0 if it is not.",
```

Comment:
There can be many builds for a single `Module` CR. Should we also add e.g. the kernel version as a label?

Reply:
fixed